### PR TITLE
Check for creator Stripe account/touchup purchase strings/bugfixes

### PR DESCRIPTION
**Pull request by mayeaux**

### CODE REVIEW

File: ui/component/preorderContent/view.jsx [hunk 1]
```diff
@@ -16,13 +15,13 @@ const STRINGS = {
...
     button: 'Purchase your content for %currency%%amount%',
-    add_card: '%add_a_card% to purchase content.',
+    add_card: '%add_a_card% to purchase content',
   },
   preorder: {
     title: 'Pre-order Your Content',
     subtitle: 'This content is not available yet but you can pre-order it now so you can access it as soon as it goes live.',
     button: 'Pre-order your content for %currency%%amount%',
-    add_card: '%add_a_card% to preorder content.',
+    add_card: '%add_a_card% to preorder content',
```
Comment (infinite-persistence) [inline ui/component/preorderContent/view.jsx:24]: Make the matching change in `app-string`

File: ui/component/preorderContent/view.jsx [hunk 2]
```diff
@@ -153,7 +132,8 @@ export default function PreorderContent(props: Props) {
       userParams,
       claimId,
       stripeEnvironment,
-      preferredCurrency,
+      'USD', // hardcode to USD for the moment
```
Comment (tzarebczan) [inline ui/component/preorderContent/view.jsx:135]: Why?

**Comment by tzarebczan** [comment]
Can you also squash once reviewed so it's re-basable.

File: ui/component/preorderContent/view.jsx (hunk 2 above)
Comment (tzarebczan) [reply ui/component/preorderContent/view.jsx:135]: I'd wanna revert this and keep preferred...would be confusing to have the setting but not use it here. Even easier that they are 1:1.

**Pull request closed without merging**